Dockerfile

```diff
@@ -1,4 +1,35 @@
-FROM node:21.7.3-bookworm-slim
+FROM node:21.7.3-bookworm-slim as base
+
+# Prerequisites
+
+# renovate: datasource=repology depName=debian_12/ca-certificates versioning=loose
+ENV CACERTIFICATES_VERSION=20230311
+
+# Ca-Certificates is required for connection to Azure DevOps
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends ca-certificates=${CACERTIFICATES_VERSION} && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+FROM base as Build
+
+# Make sure to fail due to an error at any stage in shell pipes
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# renovate: datasource=repology depName=debian_12/curl versioning=deb
+ENV CURL_VERSION=7.88.1-10+deb12u5
+
+# renovate: datasource=github-releases depName=fluxcd/flux2 extractVersion=^v(?<version>.*)$
+ENV FLUX_VERSION=2.2.3
```

Comment:
Nitpick: In other places we have the `ENV` variables directly before the command which uses them and not together at the beginning. To be consistent it would be nice to have it moved in front of line 29.